### DISCUSSION TITLE
无法识别

### DISCUSSION THREAD

**Discussion by micro-hao** [Meloong-Git/PCL#5218] [提问]
在尝试将整合包拖入pcl2时出现无法识别现象怎么办
[神秘岛永夜RPG生存地图.zip](https://github.com/user-attachments/files/18016294/RPG.zip)
[Log5.txt](https://github.com/user-attachments/files/18016298/Log5.txt)

**Comment by LinQingYuu** [comment]
>在尝试将整合包拖入pcl2时出现无法识别现象怎么办

这是存档不是整合包...
支持导入存档见 #388 ，已被拒绝
